src/HotChocolate/Stitching/src/Stitching/DependencyInjection/HotChocolateStitchingRequestExecutorExtensions.cs

```diff
@@ -571,6 +573,71 @@ await loadSchema(services, cancellationToken)
                 });
         }
 
+        /// <summary>
+        /// Adds a schema SDL that contains type extensions.
+        /// Extension documents can be used to extend merged types
+        /// or even replace them.
+        /// </summary>
+        /// <param name="builder">
+        /// The <see cref="IRequestExecutorBuilder"/>.
+        /// </param>
+        /// <param name="assembly">
+        /// The assembly from which the type extension file shall be resolved.
+        /// </param>
+        /// <param name="key">
+        /// The resource key of the type extension file
+        /// </param>
+        /// <returns>
+        /// Returns the <see cref="IRequestExecutorBuilder"/>.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="builder"/> is <c>null</c>, or
+        /// <paramref name="assembly"/> is <c>null</c>.
+        /// <paramref name="key"/> is <c>null</c>.
+        /// </exception>
+        public static IRequestExecutorBuilder AddTypeExtensionsFromResource(
```

Comment:
I think for sake of completness we should split this method into two and make also an extension on the schema builder

Reply:
???? really ???? this is schema stitching nothing of this works on schema builder.

Reply:
fair enough

Reply:
AddTypeExtensions is another extension method that is only available in stitching and stores it in the context so that the stitching schema interceptor can copy it into the stitching type interceptor which will merge and rewrite the type structure. This has nothing to do with the schema builder.